packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -11,6 +11,9 @@ import { Query } from './gql-Query'
 import { ScaffoldedFile } from './gql-ScaffoldedFile'
 import { WIZARD_BUNDLERS, WIZARD_FRAMEWORKS } from '@packages/scaffold-config'
 import { ErrorTypeEnum } from '../enumTypes'
+import debugLib from 'debug'
+
+const debug = debugLib('cypress:graphql:mutation')
```

Comment:
is this being used?

Reply:
Yes it is used here https://github.com/cypress-io/cypress/blob/develop/packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts#L668